docs/content/guides/dagster/branch-depl-with-snowflake.mdx

```diff
@@ -0,0 +1,348 @@
+---
+title: Dagster Cloud Branch Deployments with Snowflake | Dagster
+description: This guide illustrates a workflow that enables testing Dagster code in your cloud environment without impacting your production data.
+---
+
+# Dagster Cloud Branch Deployments with Snowflake
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+This guide details a workflow to test Dagster code in your cloud environment without impacting your production data. To highlight this functionality, we’ll leverage Dagster Cloud branch deployments and a Snowflake database to:
+
+- Execute code on a feature branch directly on Dagster Cloud
+- Read and write to a unique per-branch clone of our Snowflake data
+
+With these tools, we can merge changes with confidence in the impact on our data platform and with assurance that our code will execute as intended.
+
+If you’re new to Dagster, here’s an overview of the main concepts we’ll be using:
```

Comment:
with the switch to defining what the concepts will do, I think prefacing this with "If you're new to dagster" is unnecessary. 
```suggestion
Here’s an overview of the main concepts we’ll be using:
```